doc/airline.txt

```diff
@@ -785,6 +785,27 @@ localsearch <https://github.com/mox-mox/localsearch>
 * enable/disable localsearch indicator integration >
   let g:airline#extensions#localsearch#enabled = 1
 
+-------------------------------------                    *airline-ale*
+lsp <https://github.com/prabirshrestha/vim-lsp>
+
+* enable/disable lsp integration >
+  let g:airline#extensions#lsp#enabled = 1
+
+* ale error_symbol >
```

Comment:
typo.  it may be right lsp, too

Reply:
also, see the other mentions of `ale` in the help doc